docs/eventing/channel-based-broker.md

```diff
@@ -134,21 +134,23 @@ installed with Knative Eventing
 The `ServiceAccount` was created two commands prior. The `RoleBinding` is
 created with this command.
 
-Create RBAC permissions granting access to shared configmaps for logging,
-tracing, and metrics configuration.
-
-_These commands assume the shared Knative Eventing components are installed in
-the `knative-eventing` namespace. If you installed the shared Knative Eventing
-components in a different namespace, replace `knative-eventing` with the name of
-that namespace._
-
+The broker expects 3 configmaps to run properly: `eventing-config-logging`, `eventing-config-observability`
+and `eventing-config-tracing`. You can copy the corresponding configmaps under [`eventing/config/`](https://github.com/knative/eventing/tree/master/config).
```

Comment:
This should probably not reference the master branch, since that will diverge over time from the required configs for this version.

Reply:
I used the latest release branch. Let me know if that works.